README.md

```diff
@@ -1,380 +1,3 @@
-
-Depending on the limitations of the project (e.g. a mobile application with a
-limited connection, Sentry hosted on an external network), it can be useful
-to compress the data beforehand or not.
-
-It's possible to manually enable/disable the compression with the option
-`raven.compression`
-
-    http://public:private@host:port/1?raven.compression=false
-
-### Max message size
-By default only the first 1000 characters of a message will be sent to
-the server. This can be changed with the `raven.maxmessagelength` option.
-
-    http://public:private@host:port/1?raven.maxmessagelength=1500
-
-### Timeout (advanced)
-A timeout is set to avoid blocking Raven threads because establishing a
-connection is taking too long.
-
-It's possible to manually set the timeout length with `raven.timeout`
-(in milliseconds):
-
-    http://public:private@host:port/1?raven.timeout=10000
-
-## Custom RavenFactory
-At times, you may require custom functionality that is not included in `raven-java`
-already. The most common way to do this is to create your own RavenFactory instance
-as seen in the example below.
-
-```java
-public class MyRavenFactory extends DefaultRavenFactory {
-
-    @Override
-    public Raven createRavenInstance(Dsn dsn) {
-        Raven raven = new Raven();
-        raven.setConnection(createConnection(dsn));
-
-        /*
-        Create and use the ForwardedAddressResolver, which will use the
-        X-FORWARDED-FOR header for the remote address if it exists.
-         */
-        ForwardedAddressResolver forwardedAddressResolver = new ForwardedAddressResolver();
-        raven.addBuilderHelper(new HttpEventBuilderHelper(forwardedAddressResolver));
-
-        return raven;
-    }
-
-}
-```
-
-### Registration
-Next, you'll need to make your class known to Raven in one of two ways.
-
-#### Java ServiceLoader provider (recommended)
-You'll need to add a `ServiceLoader` provider file to your project at
-`src/main/resources/META-INF/services/com.getsentry.raven.RavenFactory` that contains
-the name of your class so that it will be considered as a candidate `RavenFactory`. For an example, see
-[how we configure the DefaultRavenFactory itself](https://github.com/getsentry/raven-java/blob/master/raven/src/main/resources/META-INF/services/com.getsentry.raven.RavenFactory).
-
-#### Manual registration
-You can also manually register your `RavenFactory` instance. Note that this should be done
-early in your application lifecycle so that your factory is available the first time
-you attempt to send an event to the Sentry server.
-```java
-class MyApp {
-    public static void main(String[] args) {
-        RavenFactory.registerFactory(new MyRavenFactory());
-        // ... your app code ...
-    }
-}
-```
-
-### Configuration
-Finally, see the `README` for the logger integration you use to find out how to
-configure it to use your custom `RavenFactory`.
+See the [Sentry documentation](https://docs.sentry.io/clients/java/) for more information.
```

Comment:
I think it'd be helpful to provide a more thorough introduction here at the root since this is the first thing you're going to see when you view the repository. Otherwise this looks unfinished, IMO.